chapters/BP_016_en.md

```diff
@@ -0,0 +1,39 @@
+## Use performance-oriented application forks
+
+### Identifiers
+
+| GreenIT |  V2  |  V3  |  V4  |
+|:-------:|:----:|:----:|:----:|
+|  4    | 53  | 16  |      |
+
+### Categories
+
+| Life cycle |  Tiers  |  Responsible  |
+|:---------:|:----:|:----:|
+| 3. Implementation | Datacenter | System administrator |
+
+### Indications
+
+| Priority |      Implementation difficulty       |  Ecological impact    |
+|:-------------------:|:-------------------------:|:---------------------:|
+| 3 | 3 | 3 |
+
+|Saved resources                                    |
+|:----------------------------------------------------------:|
+|  Processor / RAM / Network  |
+
+### Description
+
+Open source softwares are often forked to improve performance. Performance gains generally imply resource consumption shrinkage. Therefore if an optimized fork already exists and offers sufficient functional and technical scopes for your projects, you should therefore use it.
+
+### Example
+
+ - Prefer the optimized Pressflow version over Drupal.
```

Comment:
Either use « Prefer the optimized Pressflow over Drupal. » or « Prefer Pressflow, an optimized  version of Drupal. ». idem for the redis example.